util/chunk/column.go

```diff
@@ -63,15 +63,25 @@ type Column struct {
 	offsets    []int64
 	data       []byte
 	elemBuf    []byte
+	ft         types.FieldType
 }
 
 // NewColumn creates a new column with the specific length and capacity.
 func NewColumn(ft *types.FieldType, cap int) *Column {
+	var col *Column
 	typeSize := getFixedLen(ft)
 	if typeSize == varElemLen {
-		return newVarLenColumn(cap, nil)
+		col = newVarLenColumn(cap, nil)
+	} else {
+		col = newFixedLenColumn(typeSize, cap)
 	}
-	return newFixedLenColumn(typeSize, cap)
+	col.ft = *ft
+	return col
+}
+
+// Type returns the type of data stored in this Column.
+func (c *Column) Type() *types.FieldType {
```

Comment:
What is this method used for?

Reply:
I think we don't need to store FieldType in Column which makes the struct larger.

Reply:
The vectorized evaluation method for `Expression` may be defined as `VecEval(ctx Ctx, sel []int, result *Column)` and you can see there is no place to show what type of data the caller wants.
Therefore we add the `Type` method to `Column` to indicate what type of data should be in the `result`.

Reply:
If we remove this method, we have to implement a `VecEvalType` for each type or add a new argument to it like `VecEval(ctx, sel, result, type)`.
What's your opinion?

Reply:
Anyway, I removed this `Type()` method.
Let us discuss this problem in the next PR 🐒

Reply:
My fault, after discussing with @coocood , I think we don't need this method any more...